website/content/docs/discovery/dns.mdx

```diff
@@ -488,18 +488,17 @@ fields must be present:
 * `partition`
 * `datacenter`
 
-For node lookups, only the partition and datacenter need to be specified as nodes cannot be
-namespaced.
+For imported lookups, only the namespace and peer need to be specified as the partition that is peered can be inferred:
 
 ```text
-[tag.]<service>.service.<partition>.ap.<datacenter>.dc.<domain>
+<service>.virtual[.namespace][.peer].<domain>
 ```
 
-For node lookups, only the partition and datacenter need to be specified (nodes cannot be
```

Comment:
The diff is misleading; there were similarly worded sections for node (which had the wrong copy-pasted example)